rllib/models/preprocessors.py

```diff
@@ -106,7 +106,7 @@ def _init_shape(self, obs_space, options):
         return shape
 
     @override(Preprocessor)
-    def transform(self, observation):
+    def transform(self, observation: Any) -> np.ndarray:
```

Comment:
Shouldn't this be TensorType?

Reply:
The base class PreProcessor has np.ndarray as typing. I am assuming this applies to all other subclasses

Reply:
@sven1977